kiss_slam/kiss_slam_pybind/occupancy_mapper/occupancy_mapper.cpp

```diff
@@ -69,6 +69,18 @@ std::tuple<Vector3iVector, std::vector<float>> OccupancyMapper::GetOccupancyInfo
     return std::make_tuple(voxel_indices, voxel_occupancies);
 }
 
+Vector3iVector OccupancyMapper::GetOccupiedVoxels(float probability_threshold) const {
+    Vector3iVector voxel_indices;
+    const auto num_of_active_voxels = map_.activeCellsCount();
+    voxel_indices.reserve(num_of_active_voxels);
+    map_.forEachCell([&](const float &logodds, const Bonxai::CoordT &voxel) {
+        if (ProbabilityOccupied(logodds) > probability_threshold) {
```

Comment:
You can instead convert the `probability_threshold` into its logodds value once and do the comparison in terms of logodds. This would avoid repeated calls of `ProbabilityOccupied` to perform the thresholding for each voxel.